src/seedu/addressbook/commands/totalCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+/**
```

Comment:
Please change the name to TotalCommand.java, I think the current version cannot compile.